### ISSUE TITLE
Failed to Install

### ISSUE THREAD

**Issue by yeltsin** [adamcooke/staytus#129]
Hello.

I tried to set up a new server at Digital Ocean with Ubuntu 16.04 LTS without any "1-click-apps". I followed the tutorial of how to install Ruby with rbenv here: [How To Install Ruby on Rails with rbenv on Ubuntu 16.04](https://www.digitalocean.com/community/tutorials/how-to-install-ruby-on-rails-with-rbenv-on-ubuntu-16-04)

My ruby version is:

```bash
root@status:~# ruby -v
ruby 2.3.3p222 (2016-11-21 revision 56859) [x86_64-linux]
```

I also installed the MySQL, Bundle and RubyGems. You can check the version here:

```bash
root@status:~# bundle -v
Bundler version 1.14.5
root@status:~# rails -v
Rails 5.0.2
root@status:~# gem -v
2.5.2
```

Since the system requirement is for **Ruby 2.1 or greater**, I used ruby 2.3.3.

My server is a $5/m DO (512MB RAM). 

I tried installing staytus using the installation guide. This is where I have the problem:

```bash
bundle install --deployment --without development:test
```

This is the error:

**An error occurred while installing mysql2 (0.3.18), and Bundler cannot
continue. Make sure that `gem install mysql2 -v '0.3.18'` succeeds before bundling.**

I tried to do the "gem install mysql2 -v '0.3.18" and...

```bash
ERROR:  Error installing mysql2:
        ERROR: Failed to build gem native extension.
```

Don't know if it is a error in mysql2 or in staytus. If anyone had this same problem, please share with us (me) so I can try to fix it. 😄 

PS: If you need any more information about this problem, I can post the error.log here.

**Comment by meteormanaged** [comment]
> I also installed the MySQL, Bundle and RubyGems.

In the following lines, you don't show that you've installed MySQL gem but the version for rails, gem itself and bundle.

> I tried to do the "gem install mysql2 -v '0.3.18" and...

It looks like the failure is here with the dependency rather than with staytus itself.

**Comment by meteormanaged** [comment]
Give this a try

`sudo apt-get install build-essential libmysqlclient-dev`

**Comment by Jachim208** [comment]
Hi @meteormanaged @yeltsin 
Did you manage to solve the issue? 
I came across the same problem. So far I tried, `sudo apt-get install build-essential libmysqlclient-dev`
and a lot of ideas from the internet, including `install default-libmysqlclient-dev`, reinstalling ubuntu completly, and nothing.
It always get's stuck on `gem install mysql2`
Any ideas?
![VirtualBoxVM_oHER8ffaGL](https://user-images.githubusercontent.com/58510336/70134480-786d1700-1688-11ea-9755-4bd003ae3f15.png)
